### ISSUE TITLE
[Bug]: Sample bootstrap css files no longer work with BTCPay Point of Sale Apps without modification

### ISSUE THREAD

**Issue by Liongrass** [btcpayserver/btcpayserver#5720]
### What is your BTCPay version?

I'm running BTCPay Server v1.12.4+2111b67e2

### How did you deploy BTCPay Server?

Lunanode

### What happened?

I have always appreciated the feature that lets me take almost any bootstrap CSS file from the web and plug it into my BTCPay stores. For example [this sample shop](https://bitcoiners.encrypt.cash/) and [this sample PoS](https://we.encrypt.cash/apps/2Tq2HKCX5WCAeecjuCdZWpVZWJg2/pos) both use [this css file](https://bootswatch.com/5/vapor/bootstrap.min.css). I didn't even have to host the css itself, I just copy it from [sites like these](https://bootswatch.com/).

Recently, I noticed that a lot of these bootstrap CSS files no longer work.

Here are some, that when plugged into the PoS app, don't show the numpad or the tipping buttons, for example:
https://bootswatch.com/5/lux/bootstrap.min.css
https://bootswatch.com/5/flatly/bootstrap.min.css
![Screenshot 2024-02-02 at 15-01-58 Container Brewing](https://github.com/btcpayserver/btcpayserver/assets/2104935/c5abac66-cefa-4359-8e28-497b378ecc5f)
![Screenshot 2024-02-02 at 15-00-41 Container Brewing](https://github.com/btcpayserver/btcpayserver/assets/2104935/9db3c858-8aaf-4a6b-9a42-06cb1aaf53f5)
![Screenshot 2024-02-02 at 14-39-25 Container Brewing](https://github.com/btcpayserver/btcpayserver/assets/2104935/5900cdaf-d1b1-4d15-b682-225ba1b361aa)

### How did you encounter this bug?

1. Try out a lot of sample bootstrap CSS files
2. Add them to my Point of Sale under "Custom CSS"
3. Save and view the app

### What browser do you use?

I tried opening these sites in Firefox and Chrome.

### Additional information

In short, I believe I was once able to take almost any sample bootstrap CSS and use it right away without editing it, but this doesn't work with the PoS as of today.

### Are you sure this is a bug report?

- [X] I confirm this is a bug report

**Comment by Liongrass** [comment]
One thing to add: This might also have to do with dark mode, although some buttons/text does not show either way.

**Comment by pavlenex** [comment]
Do we still support custom bootstrap @dennisreimann ? 

@Liongrass You can customize things easily now from our Store Settings > Branding, set up default brand colors and logo and it will reflect your POS.

**Comment by Liongrass** [comment]
Hm yeah we might have to go with that. One trouble is that it works well with light mode, but not dark mode.

**Comment by dennisreimann** [comment]
@Liongrass the brand color will work in both modes, for the logo you might need to provide a version that works in both cases, e.g. with a background color. 

@pavlenex I wouldn't say we are still supporting it, either a particular theme works or it doesn't and I wouldn't wanna introduce yet another way of adjusting colors. We also have the option to adjust the colors via CSS variables if more than defining the brand color is necessary.
